core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezones specification</b>. <br/>
+  Starting from 1.615 Jenkins supports the specification 
+  of different timezones in this field. It allows to natively configure all
+  periodic operations.
```

Comment:
I kind of understand the intent, but it does sound weird.

How about replacing both sentences with:
> Periodic tasks are normally executed at the scheduled time in the timezone of the Jenkins master JVM (currently Central European Time). This behaviour can be changed by specifying an alternative timezone here."

"Central European Time" comes from `TimeZone.getDefault().getDisplayName()`, and I say "currently" because the server timezone may change, e.g. daylight savings.

(This text isn't easy to write!)